test/integration/sagemaker/util.py

```diff
@@ -96,6 +98,11 @@ def find_or_put_model_data(region, boto_session, local_path):
 def sagemaker_endpoint(sagemaker_client, model_name, instance_type, accelerator_type=None):
     logger.info('creating endpoint %s', model_name)
 
+    # Add jitter so we can run tests in parallel without running into service side limits.
+    delay = round(random.random()*5, 3)
```

Comment:
no space around '*'... does this pass pylint/flakes checks?

Reply:
I did run flake8 locally. We might not have it setup. I will check

Reply:
flake8 and pylint only checks the container folder. I will add the test folder.

Reply:
if they aren't included now, there will probably be a lot for changes for that. we should do that in separate PR later.